src/pages/drift/index.tsx

```diff
@@ -233,6 +233,20 @@ const Drift = () => {
                         Fix Iverksettinger
                     </Knapp>
                 </div>
+                {RemoteData.isFailure(fixSøknaderResponse) && (
+                    <AlertStripe className={styles.alert} type="feil">
+                        Fix Søknader feilet
+                        {fixSøknaderResponse.error.statusCode}
```

Comment:
```suggestion
                        <p>{fixSøknaderResponse.error.statusCode}</p>
```
Eventuelt <Normaltekst ... > nevner Ramzi.